packages/eas-json/src/EasJsonReader.ts

```diff
@@ -39,10 +42,11 @@ export class EasJsonReader {
         easJson.builds?.ios || {}
       );
     }
+    const experimental = easJson.experimental ?? {};
     return {
       builds: {
         ...(androidConfig ? { android: androidConfig } : {}),
-        ...(iosConfig ? { ios: iosConfig } : {}),
+        ...(iosConfig ? { ios: { ...iosConfig, ...experimental } } : {}),
```

Comment:
What if we add another field to the `experimental` object? I'd prefer explicitly choosing all relevant keys from the object.

Reply:
Or maybe we shouldn't merge it into the ios object, and instead, return an object of a similar structure as eas.json?